Model/Service/Facet/BuildWebFacetService.php

```diff
@@ -114,6 +117,7 @@ public function getFacets(): Facet
         try {
             $this->populateFilters($includeFilters);
         } catch (Exception $e) {
+            $this->debugWithSource(sprintf('Cannot populate filters, error message: %e', $e));
             $this->logger->exception($e);
```

Comment:
Why is this using the logger directly? Now that we have the trait, shouldn't this be `$this->exception`?

Reply:
There is no exception function in the LoggerTrait only in the Logger.

Reply:
In that case, please create one.

Reply:
Done